src/lib/rest/curl.cpp

```diff
@@ -0,0 +1,335 @@
+/*
+*
+* Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Fermin Galan
+*/
+#include <semaphore.h>
+#include <errno.h>
+#include <time.h>
+#include <map>  // for curl contexts
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+#include "common/clockFunctions.h"
+#include "curl.h"
+
+
+
+/* ****************************************************************************
+*  curl context
+*/
```

Comment:
```suggestion
```

As there is no need of distinguish the set of functions that follow this mark within a bigger file, as in the original place.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3869/commits/5f663bcfa195d8da60f4c74e939e7d8c19b4bb9d